data_safe_haven/infrastructure/programs/sre/dns_sidecar.py

```diff
@@ -0,0 +1,149 @@
+from pulumi import ComponentResource, Input, Output, ResourceOptions
+from pulumi_azure_native import (
+    authorization,
+    storage,
+)
+
+from data_safe_haven.functions import seeded_uuid
+from data_safe_haven.infrastructure.components import FileShareFile, FileShareFileProps
+from data_safe_haven.resources import resources_path
+from data_safe_haven.utility import FileReader
+
+# Configuration for the DNS Sidecar container.
+CONTAINER_IMAGE: str = "mcr.microsoft.com/azure-cli:latest"
```

Comment:
We prefer to pin to a specific version to ensure that we can control and test updates before they are applied to a running system.